Optimize file-type content modules (CQA) (#5056)
theforeman/foreman-documentation: guides/common/modules/proc_creating-a-local-source-for-a-custom-file-type-repository.adoc

= Creating a local source for a {customfiletype} repository

[role="_abstract"]

> You can prepare a directory on {ProjectServer} as a Pulp Manifest source and synchronize files into a {customfiletype} repository, which is useful when you have multiple files to import.

Reviewer: Again, I would split the sentences. Here, the second sentence might be a better lead sentence:

```suggestion
When you have multiple files to import, you can create a local source.
You can do so by preparing a directory on {ProjectServer} as a Pulp Manifest source and synchronizing files into a {customfiletype} repository.
```
